src/ubuntu/build-scripts/build-rootfs.sh

```diff
@@ -59,17 +59,17 @@ do
         docker rm -f $buildRootFSContainer
         exit 1
     fi
+done
 
-    echo "Tarring rootfs"
-    docker exec $buildRootFSContainer \
-        tar Ccf /rootfs - . >> $PWD/rootfs.tar
+echo "Tarring rootfs"
+docker exec $buildRootFSContainer \
+    tar -zcf $PWD/rootfs.tar /rootfs
```

Comment:
This will not work as the rpath in the tar will include rootfs.

So the rootfs will end up in /crossrootfs/rootfs/$arch

Reply:
It will also leave it in the container file system

Reply:
The original was piping the tar through stdout.  This was similar to the traditional tar pipe

`tar -C srcdir -c -f  -  . | tar -C destdir -x -f - .`
`-C` changes the directory.  
`-` represents stdout & stdin respectively
`.` represent the relative path to tar.  It is relative to the srcdir, but the srcdir path will not be in the tar stream.

Reply:
I did not bother to compress because it just would add cpu overhead

Reply:
Makes sense, I will make the changes.

Reply:
You should also move the git clone out of the loop